jaxb-ri/docs/release-documentation/src/docbook/users-guide-deployment-maven-coordinates.xml

```diff
@@ -122,16 +122,50 @@
         <title>Using &binding.impl.name; tools for java sources and XML schema generation</title>
 
         <para>
-            To generate &binding.spec.name; classes from schema in Maven project, a community
-            <link xlink:href="https://github.com/highsource/maven-jaxb2-plugin">maven-jaxb2-plugin</link> can be used.
-            <example><title>Using maven-jaxb2-plugin</title>
+            To generate &binding.spec.name; classes from schema in Maven project, there are two community plugins
+            available:
+            <itemizedlist>
+                <listitem>
+                    <para>A Maven plugin originating from MojoHaus which has been updated for JAXB 3 by Evolved Binary:
+                        <link xlink:href="https://github.com/evolvedbinary/mojohaus-jaxb-maven-plugin">MojoHaus jaxb-maven-plugin</link>
+                    </para>
+                </listitem>
+                <listitem>
+                    <para>A Maven plugin originating from java.net which has been updated for JAXB 3 by Evolved Binary:
+                        <link xlink:href="https://github.com/evolvedbinary/jvnet-jaxb-maven-plugin">jvnet jaxb-maven-plugin</link>
+                    </para>
+                </listitem>
+            </itemizedlist>
+            <example><title>Using MojoHaus jaxb-maven-plugin</title>
 
                 <programlisting language="xml"><![CDATA[
                 <build>
                     <plugins>
                         <plugin>
-                            <groupId>org.jvnet.jaxb2.maven2</groupId>
-                            <artifactId>maven-jaxb2-plugin</artifactId>
+                            <groupId>com.evolvedbinary.maven.mojohaus</groupId>
+                            <artifactId>jaxb-maven-plugin</artifactId>
+                            <executions>
+                                <execution>
+                                    <id>schemagen</id>
+                                    <goals>
+                                        <goal>schemagen</goal>
+                                    </goals>
+                                </execution>
+                            </executions>
+                        </plugin>
+                    </plugins>
+                </build>
+    ]]></programlisting>
+            </example>
+            See the <link xlink:href="https://evolvedbinary.github.io/mojohaus-jaxb-maven-plugin/">MojoHaus jaxb-maven-plugin documentation</link> for configuration details.
+
+        <example><title>Using jvnet jaxb-maven-plugin</title>
+            <programlisting language="xml"><![CDATA[
+                <build>
+                    <plugins>
+                        <plugin>
+                            <groupId>com.evolvedbinary.maven.jvnet</groupId>
+                            <artifactId>jaxb-maven-plugin</artifactId>
```

Comment:
this looks wrong, shouldn't this part be about `org.jvnet.jaxb2.maven2:maven-jaxb2-plugin` instead?

Reply:
It think the evolvedbinary project changed the name of the original jvnet plugin to leave out jaxb version and to call it a `maven-plugin` rather than a `jaxb2-plugin`.

Reply:
I am now the primary maintainer of the org.jvnet.jab2.maven2 plugin. Sadly, the original author and maintainer (Aleksei aka highsource) passed away. I just released a version for 2.x and am preparing the changes to include JAXB 3.x support.

Please do not remove the reference to Aleksei's plugin from the documentation. It is used by a ton of projects and still gets a ton of daily traffic to the GH page.

ref: https://github.com/highsource/maven-jaxb2-plugin